core/src/services/hdfs/backend.rs

```diff
@@ -244,21 +247,21 @@ impl Access for HdfsBackend {
     type BlockingDeleter = oio::OneShotDeleter<HdfsDeleter>;
 
     fn info(&self) -> Arc<AccessorInfo> {
-        self.info.clone()
+        self.core.info.clone()
```

Comment:
minor: I prefer `Arc::clone` to update the reference counter compared to the implicit `.clone()`. I was thinking about `Arc::clone(self.core.info)`.